src/tests/ftest/util/dfuse_utils.py

```diff
@@ -38,6 +38,7 @@ def __init__(self, namespace, command, path=""):
         self.disable_wb_cache = FormattedParameter("--disable-wb-cache", False)
         self.multi_user = FormattedParameter("--multi-user", False)
         self.read_only = FormattedParameter("--read-only", False)
+        self.local_flock = FormattedParameter("--enable-local-flock", False)
```

Comment:
Nit - I'd recommend making the variable match the command line more closely so there is less guesswork
```suggestion
        self.enable_local_flock = FormattedParameter("--enable-local-flock", False)
```